src/ralph/data_center/apps.py

```diff
@@ -0,0 +1,9 @@
+from ralph.apps import RalphAppConfig
+
+
+class DataCenterConfig(RalphAppConfig):
+    name = 'ralph.data_center'
+    verbose_name = 'Data Center Assets'
+
+    def get_proxies(self):
+        return ['DCHost']
```

Comment:
Ok, so that every time I added the proxy model gonna have to remember, "hmm whether definitely added model to apps.py file" wtf?

get_proxies? what is this? 

Documentation?

Permissions have to work automatically for all ralph models/extra views, and so we give a special exception for which each time will have to keep in mind.